packages/components/src/tabs/test/index.tsx

```diff
@@ -795,6 +799,57 @@ describe( 'Tabs', () => {
 			} );
 		} );
 
+		describe( 'When `selectOnMove` is `true`', () => {
+			it( 'should automatically select a newly focused tab', async () => {
+				render( <UncontrolledTabs tabs={ TABS } /> );
+
+				// Tab should focus the currently selected tab, which is Alpha.
+				await press.Tab();
+				expect( await getSelectedTab() ).toHaveTextContent( 'Alpha' );
+				expect( await getSelectedTab() ).toHaveFocus();
+
+				// Arrow keys should select and move focus to the next tab.
+				await press.ArrowRight();
+				expect( await getSelectedTab() ).toHaveTextContent( 'Beta' );
+				expect( await getSelectedTab() ).toHaveFocus();
+			} );
+		} );
+
+		describe( 'When `selectOnMove` is `false`', () => {
+			it( 'should apply focus without automatically changing the selected tab', async () => {
+				render(
+					<UncontrolledTabs tabs={ TABS } selectOnMove={ false } />
+				);
+
+				// Tab should focus the currently selected tab, which is Alpha.
+				await press.Tab();
+				expect( await getSelectedTab() ).toHaveTextContent( 'Alpha' );
+				expect( await getSelectedTab() ).toHaveFocus();
+
+				// Arrow key should move focus but not automatically change the selected tab.
+				await press.ArrowRight();
+				expect(
+					screen.getByRole( 'tab', { name: 'Beta' } )
+				).toHaveFocus();
+				expect( await getSelectedTab() ).toHaveTextContent( 'Alpha' );
+
+				// Pressing the spacebar should select the focused tab.
+				await press.Space();
+				expect( await getSelectedTab() ).toHaveTextContent( 'Beta' );
+
+				// Arrow key should move focus but not automatically change the selected tab.
+				await press.ArrowRight();
+				expect(
+					screen.getByRole( 'tab', { name: 'Gamma' } )
+				).toHaveFocus();
+				expect( await getSelectedTab() ).toHaveTextContent( 'Beta' );
+
+				// Pressing the enter/return should select the focused tab.
+				await press.Enter();
+				expect( await getSelectedTab() ).toHaveTextContent( 'Gamma' );
+			} );
+		} );
+
```

Comment:
These tests are a bit redundant:

- `selectOnMove={false}` with the uncontrolled version is already tested in the "'switches to manual tab activation when the `selectOnMove` prop is set to `false`" test
- `selectOnMove={true}`  with the uncontrolled version is kind of the default behaviour that is tested already in every test inside the "Tab Activation" section

Reply:
Shoot, I'd noticed the redundancy of that first one but totally forgot about it when pushing the last set of changes.

You make a good point on the second one being covered in multiple other tests as well. Both are removed.